docs/docs/concepts/introduction.mdx

```diff
@@ -2,43 +2,9 @@
 title: Introduction
```

Comment:
I see we have content on category pages. Should we consistently move introducing content there? So in this case /docs/concepts/introduction to /docs/concepts?

Reply:
I've actually tried it out for the pages which were lacking an overview. I think it's an good idea. We can do this as a follow-up.